cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -103,7 +102,7 @@ public static ClusterOperatorConfig fromMap(Map<String, String> map) {
      * @param lookup KafkaVersion.Lookup instance with the supported Kafka version information
      * @return  Cluster Operator configuration instance
      */
-    static ClusterOperatorConfig fromMap(Map<String, String> map, KafkaVersion.Lookup lookup) {
+    public static ClusterOperatorConfig fromMap(Map<String, String> map, KafkaVersion.Lookup lookup) {
```

Comment:
Why did we needed to make this public? I don't see it being used anywhere else in the code apart from tests.

Reply:
It is used in `io/strimzi/operator/cluster/operator/assembly/ConnectorMockTest.java` and unless that method is public that Test class cannot use it AFAIK?